src/Test/Utilities/Portable/MarkedSource/MarkupTestFile.cs

```diff
@@ -228,7 +228,7 @@ public static void GetSpans(string input, out string output, out IDictionary<str
         public static void GetPositionAndSpans(string input, out string output, out int cursorPosition, out ImmutableArray<TextSpan> spans)
         {
             GetPositionAndSpans(input, out output, out int? pos, out spans);
-            cursorPosition = pos.Value;
+            cursorPosition = pos ?? default;
```

Comment:
❗️ This seemed to be doing the correct thing before. The caller that was having a problem should be updated to instead call the overload that *optionally* returns `cursorPosition` as an `int?`.

Reply:
Fixed :)